src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java

```diff
@@ -0,0 +1,20 @@
+/*
+WhitespaceAfter
+tokens = LITERAL_TRY
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespaceafter;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class InputWhitespaceAfterLiteralTry {
+
+        public static void main(String[] args) throws IOException {
+            // violation below ''try' is not followed by whitespace'
+            try(InputStream ignored = System.in) {
```

Comment:
Put comment on same line.
Do indentation of file to be +2 to give you more space

Reply:
Changes Done